src/tools.py

```diff
@@ -173,7 +182,8 @@ def b58_decode(s):
     for c in s:
         n *= 58
         if c not in b58_digits:
-            raise InvalidBase58Error('Character %r is not a valid base58 character' % c)
+            raise InvalidBase58Error('Character %r is not a ' +
```

Comment:
I think you meant "can't". :) And I do agree that other checks are good, but not this one.